lib/workers/branch/commit.ts

```diff
@@ -47,7 +47,7 @@ export function commitFilesToBranch(
     throw new Error(CONFIG_SECRETS_EXPOSED);
   }
   // API will know whether to create new branch or not
-  return commitFiles({
+  return platform.commitFiles({
```

Comment:
We can add a `commit-files.ts` for this function, so we can easier mock in tests.
Related to comment above.

Reply:
I don't think we need this for now, all required mocks are easy enough